mlx/ops.cpp

```diff
@@ -718,6 +718,39 @@ array stack(const std::vector<array>& arrays, StreamOrDevice s /* = {} */) {
   return stack(arrays, 0, s);
 }
 
+/** array repeat with axis */
+array repeat(const array& arr, int repeats, int axis, StreamOrDevice s) {
+  axis = normalize_axis(axis, arr.ndim());
+
+  if (repeats < 0) {
+    throw std::invalid_argument("Number of repeats cannot be negative");
```

Comment:
One more nit: add "[repeat]" to the error message

Reply:
Fixed 👍